hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestRegionOpen.java

```diff
@@ -135,4 +139,36 @@ public void testNonExistentRegionReplica() throws Exception {
     }
     fail("Should have thrown IOE when attempting to open a non-existing region.");
   }
+
+  /*
+   * Before applying the fix run this test case, error message
+   * "java.io.IOException: The new max sequence id 1 is less than the old max sequence id 17"
+   * can be found the log.
+   * After applying the fix run this test case above message should not be logged.
+   */
+  //@Test(timeout = 60000)
+  @Ignore
```

Comment:
Did you mean to leave the @Ignore in place?

Reply:
I was thinking not to run that test case in the CI as it is not doing any assert, will be of no use In CI.
It was written to reproduce the issue. I think better to remove it from PR.